src/components/cms/Dashboard/MyAnalytics.js

```diff
@@ -16,30 +16,49 @@ const Container = styled.div`
   }
 `;
 
-class MyAnalytics extends Component {
-  state = {
+const MyAnalytics = props => {
+  const [state, setState] = useState({
```

Comment:
Declare separate variables for different state variables. This approach is not recommended and may cause inconsistency in case of deeply nested keys.